cmd/podmanV2/common/default.go

```diff
@@ -12,6 +12,19 @@ import (
 	"github.com/opencontainers/selinux/go-selinux"
 )
 
+var (
+	// DefaultHealthCheckInterval default value
+	DefaultHealthCheckInterval = "30s"
+	// DefaultHealthCheckRetries default value
+	DefaultHealthCheckRetries uint = 3
+	// DefaultHealthCheckStartPeriod default value
+	DefaultHealthCheckStartPeriod = "0s"
+	// DefaultHealthCheckTimeout default value
+	DefaultHealthCheckTimeout = "30s"
+	// DefaultImageVolume default value
+	DefaultImageVolume = "bind"
+)
```

Comment:
The other defaults are in the registry package. And shouldn't these be `const`?

Reply:
because they are only used in the cli, it make sense to me to leave them here.